aptos-move/e2e-testsuite/Cargo.toml

```diff
@@ -41,4 +41,8 @@ proptest = { workspace = true }
 serde = { workspace = true }
 
 [features]
-default = ["aptos-cached-packages/fuzzing"]
+default = [
+    "aptos-cached-packages/fuzzing",
```

Comment:
We don't want to turn those features on by default -- fuzzing code would end up in production. 

Unit tests are run as part of CI and they pass. See e.g. here for the CI action: https://github.com/aptos-labs/aptos-core/blob/7f9464c556709788f0f56b98a0f882702f968c18/.github/actions/rust-unit-tests/action.yaml#L41

We also run fuzzing regularily, @gedigi can say more about this. So there must be some difference in how you are trying to run things, which is not the defauilt way.

Reply:
this crate is a e2e test suite, why it will will be included in production? I guess the production build should not depend on this crate, right?

Reply:
Ah ok -- I misread the location of the toml. Can you rebase and we run CI again?

Reply:
done, please check